ee/clickhouse/queries/funnels/test/test_funnel_persons.py

```diff
@@ -166,33 +166,3 @@ def test_funnel_window_days_to_microseconds(self):
     def test_funnel_window_days_to_milliseconds(self):
         one_day = FunnelWindowDaysMixin.milliseconds_from_days(1)
         self.assertEqual(one_day, 86_400_000)
-
-    def test_basic_conversion_window(self):
```

Comment:
removed this for now. We should add some tests around conversion windows and more with step timing before merging this